app/ilmoweb/templates/open_labs.html

```diff
@@ -117,13 +117,16 @@ <h1 class="modal-title fs-5" id="studentsLabel">
           {% else %}
           <td>
             {% csrf_token %}
-            <button class="btn btn-light btn-outline-dark" onclick="signup('{{ user.id }}', '{{ lab_group.id }}','{{ lab.name }}');">Ilmoittaudu</button>
+            <button class="btn btn-light btn-outline-dark" onclick="signup(
+              '{{ user.id }}', '{{ lab_group.id }}', '{{ lab.name }}',
+              '{{ lab_group.signed_up_students }}', '{{ lab.max_students }}'
+              );">Ilmoittaudu</button>
           </td>
           {% endif %}
           {% if lab_group.status == 1 %}
             {% if user.is_staff %}
             <td>
-              {% if lab_group.signed_up_students >= lab.minim_students %}
+              {% if lab_group.signed_up_students == lab.max_students %}
```

Comment:
Staff member should be able to confirm lab-group even if the number of signed up students doesn't equal max_students. Maybe like this ```{% if lab_group.signed_up_students > 0 %}```